programs/util.h

```diff
@@ -39,7 +38,7 @@ extern "C" {
 #include "mem.h"          /* U32, U64 */
 
 
-/* ************************************************************
+/*-************************************************************
```

Comment:
What is the significance of this change?

Reply:
Every other comment was starting like that. Just to make it look the same